eZ/Bundle/EzPublishCoreBundle/Resources/views/default/full.html.twig

```diff
@@ -0,0 +1,5 @@
+<h2>{{ ez_content_name(content) }}</h2>
+{% for field in content.FieldsByLanguage %}
```

Comment:
ok. was just for consistency, but if this is the reason i wouldn't change it for now.